js/src/pages/price-benchmark/change-price-modal/index.js

```diff
@@ -247,12 +259,28 @@ const ChangePriceModal = ( { productId, onRequestClose, onPriceChange } ) => {
 						<hr className="gla-change-price-modal__separator" />
 
 						{ isOnSale && (
-							<Badge intent="warning">
-								{ __(
-									'Product is currently on sale',
-									'google-listings-and-ads'
+							<Notice status="warning" isDismissible={ false }>
+								{ createInterpolateElement(
+									__(
+										'This product is currently on sale. To change the sale price, go to the <link>Edit Product</link> page in WooCommerce.',
+										'google-listings-and-ads'
+									),
+									{
+										link: (
+											<TrackableLink
+												target="_blank"
+												type="external"
```

Comment:
Is `type` supposed to be the attribute added to the link element? If so, this doesn't seem correct to me, as `type` is meant to define the mime type of the resource being linked to. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/link#type